contractcourt/commit_sweep_resolver.go

```diff
@@ -241,14 +238,6 @@ func (c *commitSweepResolver) Stop() {
 	close(c.quit)
 }
 
-// IsResolved returns true if the stored state in the resolve is fully
-// resolved. In this case the target output can be forgotten.
-//
-// NOTE: Part of the ContractResolver interface.
-func (c *commitSweepResolver) IsResolved() bool {
-	return c.resolved
```

Comment:
I think we can move this method to the contract kit, that way we hide the underlying atomic details.

Reply:
done